core/src/main/java/jenkins/model/GlobalUIConfiguration.java

```diff
@@ -0,0 +1,47 @@
+/*
+ * The MIT License
+ * 
+ * Copyright (c) 2020
+ * 
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ * 
+ * The above copyright notice and this permission notice shall be included in
+ * all copies or substantial portions of the Software.
+ * 
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+ * THE SOFTWARE.
+ */
+package jenkins.model;
+
+import hudson.Extension;
+import jenkins.model.ProjectNamingStrategy.DefaultProjectNamingStrategy;
+import net.sf.json.JSONObject;
+import org.jenkinsci.Symbol;
+import org.kohsuke.stapler.StaplerRequest;
+
+/**
+ * Configures the UI to render, allowing to enable the new UI that came out of the work of the UX SIG or to disable it, so the classic UI is rendered
+ *
+ * @since TODO
+ */
+@Extension(ordinal = 300)
+@Symbol("useNewUI")
+public class GlobalUIConfiguration extends GlobalConfiguration {
+
+    @Override
+    public boolean configure(StaplerRequest req, JSONObject json) throws FormException {
+        Jenkins j = Jenkins.get();
+        j.setUseNewUI(json.optJSONObject("useNewUI") != null);
```

Comment:
Wouldn't this always set the flag to true, if present in the form?

Reply:
Since it's a checkbox, if you don't check it, you will receive a null. Only when you check the input you receive a field in the JSON input